crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+- CFF `references`: "Reference(s) to other creative works."
+- CFF root document: The citation metadata for the software (or dataset) that the `CITATION.cff` file pertains to
+
+#### CodeMeta to CFF
+
+Understanding `citation` in CodeMeta to be the citation for the described software, the relationships are as follows:
+
+- When the `citation` is a citation of the software itself, it maps to the citation metadata provided in the root of a `CITATION.cff` file.
+- When the `citation` is a citation to another work describing the software (a "meta" work, e.g., a software paper in JOSS, a traditional paper about the software), it maps to CFF `preferred-citation`.
+- Diverging from the understanding of `citation` being the citation to the software, when the `citation` is to an unrelated work, it maps to an entry in CFF `references`
+
+#### CFF to CodeMeta
+
+The citation metadata in the root of a `CITATION.cff` file maps to CodeMeta `citation`.
+
+### `license` <=> `license`/`license-url`
+
+#### CodeMeta to CFF
+
+If the CodeMeta `license` contains an SPDX identifier (or its URL) that is recognized in CFF to be a valid value of CFF `license` (list [here](https://github.com/citation-file-format/citation-file-format/blob/main/schema.json#L516-L978)),
+it maps to `license`.
+If the CodeMeta `license` contains a URL to an license unrecognized in CFF, it maps to `license-url`.
+If the CodeMeta `license` contains a `schema:CreativeWork`, which contains a URL to the license, it maps to `license-url`.
+`schema:CreativeWork`s without URLs to the license cannot be mapped.
+
+#### CFF to CodeMeta
+
+CFF `license-url` maps to CodeMeta `license`.
+CFF `license` maps to CodeMeta `license` following the pattern `https://spdx.org/licenses/<value of CFF license>.html`.
+
+### `version`/`softwareVersion` <=> `version`
+
+#### CodeMeta to CFF
+
+Both CodeMeta `version` and CodeMeta `softwareVersion` map to CFF `version` (in the root of the `CITATION.cff` file).
+CodeMeta is ambiguous for this term (see <https://github.com/codemeta/codemeta/issues/264>), and it is unclear which value of the terms should be mapped if they differ.
+
+#### CFF to CodeMeta
+
+CFF `version` maps to both CodeMeta `version` and CodeMeta `software Version`.
+CodeMeta is ambiguous for this term (see <https://github.com/codemeta/codemeta/issues/264>).
+Until this is resolved, CFF `version` should probably be mapped to both.
+
+### `isPartOf`/`hasPart` <=> `identifiers`
+
+CFF doesn't (yet, see  https://github.com/citation-file-format/citation-file-format/issues/69#issuecomment-904546233) support these qualifiers for `identifiers` in a machine-readable way.
+`identifiers` is a list of identifiers pertaining to the software ("The identifiers of the software or dataset.").
+
+As of version 1.2.0, CFF `identifiers` may have an `identifier.description`, which may specify the relation of the identifier to another object,
```

Comment:
but only in human readable form

Reply:
Fixed in 992a0eb45863a5832d25dd186aa91307600dd206.